pkg/eval/json_evaluator.go

```diff
@@ -274,8 +268,37 @@ func (je *JSONEvaluator) evaluateVariant(
 	return je.state.Flags[flagKey].DefaultVariant, reason, nil
 }
 
+// configToFlags convert string configurations to flags and store them to pointer newFLags
+func (je *JSONEvaluator) configToFlags(config string, newFlags *Flags) error {
+	schemaLoader := gojsonschema.NewStringLoader(schema.FlagdDefinitions)
+	flagStringLoader := gojsonschema.NewStringLoader(config)
+	result, err := gojsonschema.Validate(schemaLoader, flagStringLoader)
+
+	if err != nil {
+		return err
+	} else if !result.Valid() {
+		err := errors.New("invalid JSON file")
+		return err
+	}
+
+	transposedConfig, err := je.transposeEvaluators(config)
+	if err != nil {
+		return fmt.Errorf("transpose evaluators: %w", err)
+	}
+
+	err = json.Unmarshal([]byte(transposedConfig), &newFlags)
+	if err != nil {
+		return fmt.Errorf("unmarshal new state: %w", err)
```

Comment:
I think these messages could be improved. Besides the fact that it probably outputs to stderr, it's not clear that it's an error. I think I'd prefer something like "unable to transpose evaluators..."

Reply:
No, this is not writing to stderr. `fmt.Errorf` generates and error, with the formatted error message [1] :) .

Sure, I will improve the error message here. I moved this code and did not change the logic, but given I am touching this, it makes sense to improve it 

[1] - https://pkg.go.dev/fmt#Errorf